src/layouts/Root/components/Header.tsx

```diff
@@ -0,0 +1,337 @@
+import { MediaInput } from '@zendeskgarden/react-forms';
+import { ReactComponent as MenuStroke } from '@zendeskgarden/svg-icons/src/16/menu-stroke.svg';
+import { ReactComponent as SearchStroke } from '@zendeskgarden/svg-icons/src/16/search-stroke.svg';
+import { ReactComponent as CloseStroke } from '@zendeskgarden/svg-icons/src/16/x-stroke.svg';
+import { ReactComponent as GardenIcon } from '../../../images/garden-icon.svg';
+import { ReactComponent as GardenWordmark } from '../../../images/garden-wordmark.svg';
+import MaxWidthLayout from 'layouts/MaxWidth';
+
+const StyledDesktopNavItem = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const StyledDesktopNavLink = styled(Link)`
+  display: flex;
+  align-items: center;
+  align-self: stretch;
+  justify-content: center;
+  padding: ${p => p.theme.space.sm};
+  line-height: ${p => p.theme.lineHeights.md};
+  color: ${p => getColor('grey', 800, p.theme)};
+  font-size: ${p => p.theme.fontSizes.md};
+`;
+
+const StyledHeader = styled.header.attrs({ role: 'banner' })`
+  z-index: 1;
+  border-bottom: ${p => p.theme.borders.sm} ${p => getColor('grey', 300, p.theme)};
+  box-shadow: ${p => p.theme.shadows.lg('0', '10px', getColor('kale', 600, p.theme, 0.15)!)};
+  padding: 0 ${p => p.theme.space.md};
+  height: ${p => p.theme.space.base * 20}px;
+
+  &[data-show-navigation='true'] {
+    border-bottom-color: ${p => p.theme.palette.white};
+  }
+
+  @media (max-width: ${p => p.theme.breakpoints.md}) {
+    padding: 0;
+    height: ${p => p.theme.space.base * 15}px;
+  }
+`;
+
+const Logo: React.FC = () => (
+  <div
+    css={css`
+      height: 100%;
+      line-height: 0;
+
+      @media (max-width: ${p => p.theme.breakpoints.md}) {
+        flex-grow: 1;
+      }
+    `}
+  >
+    <Link
+      aria-label="Zendesk Garden"
+      to="/"
+      css={`
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        height: 100%;
+      `}
+    >
+      <div>
+        <GardenIcon
+          css={css`
+            width: auto;
+            height: ${p => p.theme.lineHeights.md};
+
+            @media (max-width: ${p => p.theme.breakpoints.md}) {
+              height: ${p => p.theme.iconSizes.lg};
+            }
+          `}
+        />
+        <GardenWordmark
+          css={css`
+            margin-left: ${p => p.theme.space.xs};
+            width: auto;
+            height: ${p => p.theme.lineHeights.md};
+
+            @media (max-width: ${p => p.theme.breakpoints.md}) {
+              display: none;
+            }
+          `}
+        />
+      </div>
+    </Link>
+  </div>
+);
+
+const MobileSearch = React.forwardRef<HTMLInputElement>((props, ref) => (
+  <div
+    css={css`
+      display: flex;
+      flex-grow: 1;
+      align-items: center;
+      justify-content: center;
+      padding: ${p => p.theme.space.xxs};
+      height: 100%;
+    `}
+  >
+    <SearchInput ref={ref} isCompact={false} />
```

Comment:
Would this be the same as the following?

```
<SearchInput ref={ref} />
```

Or is passing `false` overridding something in `SearchInput`?

Reply:
Some earlier mockups had mixtures of `isCompact` and standard inputs. I've removed this prop for now.